File: core-java-practice/gcr-codebase/programming-elements/PerimeterOfRectangle.java
package Dayone;
import java.util.*;
public class PerimeterOfRectangle {
  public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        // Length of the rectangle
        int length = sc.nextInt(); 
        // Breadth of the rectangle
        int breadth = sc.nextInt(); 
        // Calculate perimeter of the rectangle
        int perimeter = 2 * (length + breadth);

        // Print the perimeter
        System.out.println("Perimeter of Rectangle is " + perimeter);
    }
}

File: core-java-practice/gcr-codebase/programming-elements/PerimeterOfRectangle.cs
using Systems;
namespace programming-elements{
  class PerimeterOfRectangle{
    static void Main(String[] args) {
        // Length of the rectangle
        int length = int.Parse(Console.ReadLine());
        // Breadth of the rectangle
        int breadth = int.Parse(Console.ReadLine());
        // Calculate perimeter of the rectangle
        int perimeter = 2 * (length + breadth);

        // Print the perimeter
        Console.WriteLine("Perimeter of Rectangle is " + perimeter);
    }
  }
}

